service/src/main/java/org/apache/griffin/core/job/JobServiceImpl.java

```diff
@@ -666,9 +669,28 @@ public void triggerJobById(Long id) throws SchedulerException {
                     .forJob(jobKey)
                     .startNow()
                     .build();
+            TriggerKey key = trigger.getKey();
+            CountDownLatch latch = new CountDownLatch(1);
+            scheduler.getListenerManager().addTriggerListener(
```

Comment:
will this listener be automatically deleted later, or should we take care of that?

Reply:
No, it doesn't happen automatically, I added it